librz/bin/format/pe/pe.c

```diff
@@ -500,9 +500,9 @@ static int bin_pe_parse_imports(struct PE_(rz_bin_pe_obj_t) * bin,
 					if (filename && rz_file_exists(filename)) {
 						db = sdb_new(NULL, filename, 0);
 					} else {
-						const char *dirPrefix = rz_sys_prefix(NULL);
-						filename = sdb_fmt(RZ_JOIN_4_PATHS("%s", RZ_SDB_FORMAT, "dll", "%s.sdb"),
-							dirPrefix, symdllname);
+						char *formats_dir = rz_path_system(RZ_SDB_FORMAT);
+						filename = sdb_fmt(RZ_JOIN_3_PATHS("%s", "dll", "%s.sdb"), formats_dir, symdllname);
```

Comment:
Same.

Reply:
Could you, please, also remove the `sdb_fmt` use while you are on it?

Reply:
+1 on sdb_fmt